<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewWindow</class>
 <widget class="QWidget" name="ViewWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1091</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../../resources/icons/buttons/buttonicons.qrc">
    <normaloff>:/icons/buttons/view.ico</normaloff>:/icons/buttons/view.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="6">
    <widget class="QLabel" name="label_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Jump row:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>From row:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Measurement ID:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="12">
    <widget class="QTableWidget" name="measurementTableWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectColumns</enum>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>160</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QSpinBox" name="toRowSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>To row:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QSpinBox" name="fromRowSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QComboBox" name="measurementIDComboBox"/>
   </item>
   <item row="0" column="7">
    <widget class="QSpinBox" name="jumpRowSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
   </item>
   <item row="0" column="13">
    <widget class="QPushButton" name="deleteDataPushButton">
     <property name="text">
      <string>Delete data</string>
     </property>
    </widget>
   </item>
   <item row="0" column="12">
    <widget class="QPushButton" name="downloadDataPushButton">
     <property name="text">
      <string>Download data</string>
     </property>
    </widget>
   </item>
   <item row="0" column="10" colspan="2">
    <widget class="QPushButton" name="plotDataPushButton">
     <property name="text">
      <string>Plot data</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../resources/icons/buttons/buttonicons.qrc"/>
  <include location="../../../../resources/icons/buttons/buttonicons.qrc"/>
  <include location="../../../../resources/icons/buttons/buttonicons.qrc"/>
 </resources>
 <connections/>
</ui>
